static/js/pages/applications/ReviewDetailPage.js

```diff
@@ -0,0 +1,254 @@
+
+import {
+  REVIEW_DETAIL_TITLE,
+  REVIEW_STATUS_APPROVED,
+  REVIEW_STATUS_PENDING,
+  REVIEW_STATUS_REJECTED,
+  REVIEW_STATUS_WAITLISTED
+} from "../../constants"
+import UserDetails from "../../components/UserDetails"
+
+import { Alert } from "reactstrap"
+
+import type { Match } from "react-router"
+
+type Props = {
+  match: Match
+}
+
+export const ReviewDetailPage = (props: Props) => {
+  const { match } = props
+  const submissionId = match.params.id
+
+  const [error, setError] = useState("")
+  const [success, setSuccess] = useState(false)
+  const [selection, setSelection] = useState("")
+
+  const submissions = useSelector(submissionDetailSelector)
+  const submission = submissions ? submissions[submissionId] : null
+
+  const applications = useSelector(applicationDetailSelector)
+  const application =
+    applications && submission ?
+      applications[submission.bootcamp_application.id] :
+      null
+  const user = submission ? submission.learner : null
+
+  useRequest(queries.applications.submissionReviewQuery(submissionId))
+  useRequest(
+    submission ?
+      queries.applications.applicationDetailQuery(
+        submission.bootcamp_application.id
+      ) :
+      null
+  )
+
+  const clearState = () => {
+    setSelection(submission ? submission.review_status : "")
+    setError()
+  }
+
+  useEffect(() => {
+    clearState()
+  }, [submission])
+
+  const [{ isPending }, updateStatus] = useMutation(status =>
+    // $FlowFixMe
+    queries.applications.submissionReviewMutation({
+      ...submission,
+      review_status: status
+    })
+  )
+
+  const onSubmit = async (status: string) => {
+    if (isNilOrBlank(status)) {
+      return
+    }
+    const result = await updateStatus(status)
+    if (result.status === 200) {
+      setSuccess(true)
+      setError()
+    } else {
+      setError(result.body.detail)
+      setSuccess(false)
+    }
+  }
+
+  const onUpdate = async (e: Object) => {
+    await setSelection(e.target.value)
+  }
+
+  const currentSelection =
+    selection || (submission ? submission.review_status : null)
+
+  return submission && user && application ? (
+    <div className="container applications-page">
+      <MetaTags>
+        <title>{formatTitle(REVIEW_DETAIL_TITLE)}</title>
+      </MetaTags>
+      <div className="row">
+        <h1 className="col-12">{REVIEW_DETAIL_TITLE}</h1>
+      </div>
+      <div className="row">
+        <div className="col-8 review-card review-panel-left">
+          <div className="row section profile-header">
+            <div className="header">{user.profile.name}</div>
+          </div>
+          {submission.interview_url ? (
+            <div className="row section interview">
+              <div className="col">
+                <div className="row">
+                  <div className="header">Video Interview</div>
+                </div>
+                <div className="row">
+                  <a
```

Comment:
Can we get rid of these wrapping elements and just put it as a child of the `row section interview` div? Same deal below for the resume and linkedin sections

Reply:
The layout gets quite messed up if I remove these wrapping elements, I think it would be best to leave as is.